ml-agents/mlagents/trainers/torch/distributions.py

```diff
@@ -122,12 +123,22 @@ def __init__(
     ):
         super().__init__()
         self.conditional_sigma = conditional_sigma
-        self.mu = nn.Linear(hidden_size, num_outputs)
+        self.mu = linear_layer(
+            hidden_size,
+            num_outputs,
+            kernel_init=Initialization.KaimingHeNormal,
```

Comment:
maybe these are the natural default vals since they seem to be common

Reply:
I tried to have the default be the same as tensorflow since it is more common